packages/twenty-ui/src/display/avatar/components/Avatar.tsx

```diff
@@ -90,7 +90,8 @@ export const Avatar = ({
       })
     : null;
 
-  const placeholderChar = placeholder?.[0]?.toLocaleUpperCase();
+  const placeholderChar = placeholder?.[0]?.toLocaleUpperCase().trim() || '-';
```

Comment:
use explicit ternary condition placeholder?.[0]?.toLocaleUpperCase().trim() === ''

Reply:
placeholder can be undefined so I tried other solutions but the orignial ones sounds the best to me